static/src/board.js

```diff
@@ -0,0 +1,54 @@
+//import './boardUtils.js';
+// Box width
+var bw = 500;
+// Box height
+var bh = 500;
+// Padding
+var p = 10;
+
+function drawGrid(ctx) {
+    ctx.beginPath();
+    for (var x = 0; x <= bw; x += 50) {
```

Comment:
We should define a const at the top of the file, instead of using 50 in so many places.